### PR TITLE
ISPN-14913 Create cache configuration

### PR DESCRIPTION
**Pull request by dpanshug**
https://issues.redhat.com/browse/ISPN-14913

### CODE REVIEW

**Comment by dpanshug** [comment]
This should be merged after #368

File: src/app/assets/languages/en.json [hunk 1]
```diff
@@ -165,6 +165,7 @@
     "setup": {
       "page-title": "Configure a cache in {{cmName}}",
       "page-title-with-backups": "Configure a cache in {{cmName}} on site {{localsite}}",
+      "page-title-description": "You can only generate a new cache config and generate it. Note that the new cache will not be stored by Infinispan",
```
Comment (karesti) [inline src/app/assets/languages/en.json:168]: Infinispan is the brand variable to be replaced by Datagrid

**Review by karesti — CHANGES_REQUESTED** [review]

**Comment by karesti** [comment]
Screenshot 2023-07-04 at 10 11 36

**Comment by karesti** [comment]
@dpanshug is there any change related to the fact of creating "templates" instead of caches ?

**Comment by karesti** [comment]
Screenshot 2023-07-04 at 10 15 31

misaligned

**Comment by karesti** [comment]
the message above, you can add the Operator as well.

**Comment by dpanshug** [comment]
> @dpanshug is there any change related to the fact of creating "templates" instead of caches ?

By change in creating "templates" do you mean creating cache config for users without create right? The only change is I've removed the cache setup from the sidebar.

**Comment by dpanshug** [comment]
> @dpanshug is this preview ?

@andyuk1986 found some warnings during the test, waiting for a final check.

**Comment by andyuk1986** [comment]
Thank you @dpanshug, the issue with exceptions in the log is fixed.

File: src/services/cacheService.ts [hunk 2]
```diff
@@ -504,6 +509,44 @@ export class CacheService {
     return customHeaders;
   }
 
+  public async convertToAllFormat(
```
Comment (karesti) [inline src/services/cacheService.ts:512]: @dpanshug  I added a method to wrap le 3 calls, and add a loading

**Comment by andyuk1986** [comment]
Tests started to fail, let me look into them.

**Comment by karesti** [comment]
@andyuk1986 I changed the code so the formatting of the config does 3 calls and waits for them with a loading
do you have time to grab a look to the e2e part f it ?

**Comment by andyuk1986** [comment]
yes, looking into it.

**Comment by karesti** [comment]
@andyuk1986 I accidentally pushed a wrong stuff! sorry wasting your time

**Pull request merged**